packages/ui-library/src/components/toggle-switch/index.ts

```diff
@@ -126,7 +126,7 @@ export class BlrToggleSwitch extends LitElementCustom {
         'disabled': this.disabled || false,
         'readonly': this.readonly || false,
         'hover': this.hovered || false,
-        'active': this.active || false,
+        'active': this.checked || false,
```

Comment:
Perhaps we should rename the class to `pressed`. The meaning of `active` can be a bit ambiguous at times.